sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/Scenario/ArmApiManagementTests.cs

```diff
@@ -0,0 +1,261 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.ApiManagement.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ApiManagement.Tests.Scenario
+{
+    public class ArmApiManagementTests : ApiManagementManagementTestBase
```

Comment:
well these new tests should be just unit test, therefore this class does not need to inherit from this base.
Let us put this in a new directory, like `tests/Unit/ModelFactoryTests.cs`